vue-components/src/components/iconProps.ts

```diff
@@ -13,6 +13,16 @@ export enum IconTypes {
 	SEARCH = 'search'
 }
 
-export const iconColors = [ 'base', 'warning', 'error', 'notice', 'success', 'inherit' ];
+export const iconColors = [
```

Comment:
This does look unrelated.

Reply:
it's probably a VSCode linting extension being "smart"

Reply:
Edit: I thought it's a VSCode extension being smart and fixing this but actually the line becomes too long hence lint fix.

Reply:
I think this will be solved once the branch is updated. For some reason those changes to the icon component are being dragged into my PRs T-T